Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -271,6 +273,10 @@ internal static string GetContentProperty(TypeReference typeRef)
 			else if (vardefref.VariableDefinition.VariableType.ImplementsGenericInterface("Xamarin.Forms.Xaml.IMarkupExtension`1",
 				out markupExtension, out genericArguments))
 			{
+				if (vardefref.VariableDefinition.VariableType.FullName == "Xamarin.Forms.Xaml.BindingExtension")
+					foreach (var instruction in CompileBindingPath(node, context, vardefref.VariableDefinition))
+						yield return instruction;
```

Comment:
this is the part that sets the `TypedBinding` property to the `BindingExtension`